src/with-renderer.js

```diff
@@ -22,14 +24,28 @@ export const withRenderer = (
       );
     }
 
+    connectedCallback() {
+      if (super.connectedCallback) {
+        // $FlowFixMe - not in HTMLElement.
+        super.connectedCallback();
+      }
+      this._connected = true;
+    }
+
     propsChangedCallback() {
+      if (super.propsChangedCallback) {
+        // $FlowFixMe - not in HTMLElement.
+        super.propsChangedCallback();
+      }
+      if (!this._connected) {
```

Comment:
This doesn't break behaviour because this is what it was explicitly intended for.